scripts/download_data.sh

```diff
@@ -70,6 +70,23 @@ if [[ "$DO_OLD" ]]; then
     DO_MR='true'
 fi
 
+# check the md5, OSX compat
+function check_md5 {
+    filename=$1
+    if [[ -e $(which md5sum) ]]; then
+        md5sum -c "${filename}.md5"
+    elif [[ -e $(which md5) ]]; then
```

Comment:
probably too much, but I don't think `which ` is going to be installed everywhere (it wasn't in `bash` a few years ago). I believe `command` should be portable. It is a [POSIX built-in of `sh` and all compliant shells](https://bs2manuals.ts.fujitsu.com/psPOSIXV100en/posix-commands-user-guide-20395/commands-posix-commands-2020-10-49/c-posix-commands-2020-10-522/command-execute-a-simple-command-posix-commands-2020-10-75)
and use in the STIR recon_test_pack, for instance [here](https://github.com/UCL/STIR/blob/40545147179fbe2b1131679672375cc91a39ab55/recon_test_pack/run_tests.sh#L89) which is tested on MacOS.

I think this could be replaced by
```suggestion
    if  command -v md5sum > /dev/null  2>&1
    then
        md5sum -c "${filename}.md5" > /dev/null  2>&1
    elif command -v md5 > /dev/null  2>&1
    then
```

Reply:
Yes, this looks better. Thanks